api/data/NMBS/liveboard.php

```diff
@@ -114,22 +114,21 @@ private static function parseData($xml, $time, $lang, $fast = false)
             $journey = $data->Journey[$i];
 
             $left = 0;
+            $canceled = false;
             $delay = (string) $journey['delay'];
             if ($delay == '-') {
                 $delay = '0';
             }
-
-            $platform = '';
-            if (isset($journey['platform'])) {
-                $platform = (string) $journey['platform'];
+            
+            if ($delay == 'cancel') {
+                $delay = 999999; // Indicate something is wrong if `canceled` is not read by the client
```

Comment:
It used to be 0. Now it will report a delay of ~27 hours. (=99999s)

> On 29 Dec 2015, at 12:23, Pieter Colpaert <notifications@github.com> wrote:
> In api/data/NMBS/liveboard.php:
> 
> >              $delay = (string) $journey['delay'];
> >              if ($delay == '-') {
> >                  $delay = '0';
> >              }
> > -
> > -            $platform = '';
> > -            if (isset($journey['platform'])) {
> > -                $platform = (string) $journey['platform'];
> > +            
> > +            if ($delay == 'cancel') {
> > +                $delay = 999999; // Indicate something is wrong if `canceled` is not read by the client
> I don't like this solution: will break certain GUIs. We should still keep '0' or 'cancel' (wasn't this how it used to be?) here...
> 
> —
> Reply to this email directly or view it on GitHub.
>